Makefile

```diff
@@ -20,14 +20,44 @@ MAN_PAGES = $(shell ls $(MAN_DIR)/*.8)
 MAN_PAGES_BASE = $(notdir $(MAN_PAGES))
 MAN_INSTALL_PATH := ${PREFIX}/share/man/man8/
 
+RELEASE_DIR := $(CURDIR)/release
+
 VERSION := ${shell cat ./VERSION}
 
+SHELL := /bin/bash
```

Comment:
I think maybe because she script is using array.

In result of "grep -w bash * -R | grep -v 'Binary file'",
we can see bash is used in scripts in following dir:
script/
tests/integration/
man/

Current runc already have some relationship with bash,
or we can say, bash is already necessary component for runc dev,
maybe we can use bash freely in current Makefile.

What is your opinion?